geektrain.html

```diff
@@ -0,0 +1,86 @@
+    <meta name="twitter:creator" content="@dotnetfringe">
+    <meta name="twitter:title" content=".NET/FRINGE/ - An atypical conference for open-source .NET developers.">
+    <meta name="twitter:description" content=".NET/FRINGE/ - An atypical conference for open-source .NET developers. June 4 - 6, 2017, Portland, OR">
+    <meta name="twitter:image" content="">
+    <!-- css: reset / layout -->
+    <link rel="stylesheet" href="http://cdn.rawgit.com/necolas/normalize.css/master/normalize.css">
+    <link rel="stylesheet" href="http://cdn.rawgit.com/milligram/milligram/master/dist/milligram.min.css">
+    <!-- css: fonts -->
+    <link href='http://fonts.googleapis.com/css?family=Lato:100,300,400,500,700,900' rel='stylesheet' type='text/css'>
+    <link href='http://fonts.googleapis.com/css?family=Righteous:100,300,400,500,700,900' rel='stylesheet' type='text/css'>
+    <!-- css: fringe theme -->
+    <link rel="stylesheet" href="css/dotnetfringe.css" />
+  </head>
+  <body>
+    <div class="container logo-container">
+      <div class="row">
+        <div class="column">
+          <h1 class="logo" data-shadow='2017'>.netfr!nge</h1>
+          <h2 class="logo"><a href="index.html"><span>JUNE 4-6, 2017 - PDX</span></a></h2>
+        </div>
+      </div>
+    </div>
+    <div class="container fullwidth content-container">
+      <div class="container">
+        <div class="row dnf-menu">
+          <div class="column">
+            <h3><a href="index.html">home</a></h3>
+          </div>
+          <div class="column">
+            <h3><a href="attend.html">attend</a></h3>
+          </div>
+          <div class="column">
+            <h3><a href="speakers.html">speakers</a></h3>
+          </div>
+          <div class="column">
+            <h3><a href="sponsors.html">sponsors</a></h3>
+          </div>
+          <div class="column">
+            <h3><a href="docs/get-involved.html">get involved</a></h3>
+          </div>
+          <div class="column">
+            <h3><a href="about.html">about</a></h3>
+          </div>
+        </div>
+        <div class="row">
+          <div class="column">
+            <h1>GEEK TRAIN!</h1>
+            <p>Everything you need to know about joining the Geek Train to .NET Fringe! First and foremost, buy your $15 buck ticket <a href="https://ti.to/dotnetfringe/dotnetfringe-2017">here</a>.</p>
+
+            <p>The train we'll be taking together is the <a href="https://www.amtrak.com/ccurl/394/275/Amtrak-Cascades-Schedule-022016,0.pdf">Amtrak Cascades #507</a> train, departing from <a href="https://en.wikipedia.org/wiki/King_Street_Station">King Street Station</a> in Seattle, Washington. If you're intending to board from a different station other than King Street Station I (<a href="https://twitter.com/Adron">@Adron</a> will contact you regarding those logistical plans, but if I haven't and you need to leave from a different station please DM/ping me ASAP.</p>
```

Comment:
In these two paragraphs you used the phrase "I" and "me" a few times, but it's not clear who you are, espcially for a first-time attendee. 

Include a recent photo of you (there is already one checked in, see the about page). Change this to third-person "Adron"... instead of "I" and "me".